.github/workflows/action.yml

```
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "14"
-      - uses: artiomtr/jest-coverage-report-action@Use-jest-config-instead-of-threshold-option
+      - uses: artiomtr/jest-coverage-report-action@test-report-threshold-failures
```

Hello @therynamo! Could you try please to change this line of code to the: 
```suggestion
      - uses: therynamo/jest-coverage-report-action@report-threshold-failures
```

---

😅 - I cannot believe I missed this 🤦 . Nice catch - and thank you! 😄